genconvert/input/my-t-my-d0-zawgyi.txt

```diff
@@ -118,6 +115,7 @@ $ukinzi ($consonant) > $1 \u1064 ;
 \u1039 \u101C > \u1085 ;
 
 \u100d\u1039\u100D > \u106E ;
+\u100d\u1039\u100E > \u106F ;
```

Comment:
IIRC, this duplicated line was causing problems.  Did you mean to remove it?

Reply:
It should be removed now.

Reply:
This should be fixed now.

Reply:
I tried updating .travis.xml as you suggested. All went well until the last step:

git pull origin master
From github.com:googlei18n/myanmar-tools
 * branch            master     -> FETCH_HEAD
Updating ec4011b..4be2b11
Fast-forward
 .travis.yml | 11 ++++++++++-
 1 file changed, 10 insertions(+), 1 deletion(-)
Current branch master is up to date.
ccornelius@ccornelius2:~/ZawgyiUnicode_GIT/GIT_source$ git checkout test26
Switched to branch 'test26'
ccornelius@ccornelius2:~/ZawgyiUnicode_GIT/GIT_source$ git merge master
Merge made by the 'recursive' strategy.
 .travis.yml | 11 ++++++++++-
 1 file changed, 10 insertions(+), 1 deletion(-)
ccornelius@ccornelius2:~/ZawgyiUnicode_GIT/GIT_source$ git status
On branch test26
nothing to commit, working tree clean
ccornelius@ccornelius2:~/ZawgyiUnicode_GIT/GIT_source$ git push
fatal: The current branch test26 has no upstream branch.
To push the current branch and set the remote as upstream, use

    git push --set-upstream origin test26

Should I do what it suggests?

Reply:
Yep, the merge commit is now pushed.  Thanks.